draft-ietf-tls-tls13.md

```diff
@@ -2444,13 +2446,20 @@ supply an "early_data" extension as well as the "pre_shared_key"
 extension.
 
 The "extension_data" field of this extension contains an
-"EarlyDataIndication" value:
+"EarlyDataIndication" value.
 
 %%% Key Exchange Messages
 
        struct {
+           select (Handshake.msg_type) {
+               case client_hello:
+               case hello_retry_request:
+               case new_session_ticket:
+                  uint32 max_early_data_size;
        } EarlyDataIndication;
```

Comment:
This is fall through. So, `uint32` is contained in `client_hello`. But the original extension is empty. How should we use `uint32` for `client_hello`?